src/meta/server_state.cpp

```diff
@@ -56,6 +56,39 @@ using namespace dsn;
 namespace dsn {
 namespace replication {
 
+DSN_DEFINE_int32("meta_server",
+                 max_allowed_replica_count,
+                 5,
+                 "max replica count allowed for any app of a cluster");
+
```

Comment:
No need to left blank lines for related DSN_TAG_VARIABLE and DSN_DEFINE_validator of a config.

Reply:
Ok